src/gap.h

```diff
@@ -209,6 +209,11 @@ extern void ErrorReturnVoid (
             Int                 arg2,
             const Char *        msg2 );
 
+typedef void  (* TJumpToCatchFunc)();
+extern TJumpToCatchFunc JumpToCatchFunc;
+static inline void SetJumpToCatchFunc(TJumpToCatchFunc func)
+{ JumpToCatchFunc = func; };
```

Comment:
That an odd code formatting style...

Also, why does this setter even exist, if `JumpToCatchFunc` is a global anyway?